outputs.tf

```diff
@@ -18,3 +18,8 @@ output "iam_role_name" {
   description = "Name of the IAM role for the NAT instance"
   value       = aws_iam_role.this.name
 }
+
+output "autoscaling_group_arm" {
+  description = "ASG ARN"
```

Comment:
"The ARN for this autoscaling group" might be a better description here based on the popular eks module https://github.com/terraform-aws-modules/terraform-aws-eks/blob/771465be280450fc96d889ef9e15f191bb512849/modules/self-managed-node-group/outputs.tf#L30